doc/install/build-ceph.rst

```diff
@@ -38,17 +38,7 @@ repository and execute the following::
     cd ceph
     ./do_cmake.sh
     cd build
-    make
-
-.. note:: By default do_cmake.sh will build a debug version of ceph that may
-   perform up to 5 times slower with certain workloads. Pass 
-   '-DCMAKE_BUILD_TYPE=RelWithDebInfo' to do_cmake.sh if you would like to
-   build a release version of the ceph executables instead.
```

Comment:
this change is not related to the ninja change. why would you want to remove it? i cannot find any explanation in the commit message.

Reply:
I removed it as `README.md` has the same note. I have created a new commit for this change and added a link to the `README` doc .